File: lib/payment_parameters.dart

class PaymentParameters{
  String cardNumber;
  String cardDate; //формат MMYY
  String cardCVV;
  String publicId;


  PaymentParameters(this.cardNumber, this.cardDate, this.cardCVV,this.publicId);

  Map<String,String> toMap(){
    var map = Map<String,String>();
    map['card_number'] = cardNumber;
    map['card_date'] = cardDate;
    map['card_cvv'] = cardCVV;
    map['public_id'] = publicId;
    return map;
  }
}

File: android/src/main/kotlin/xdev/team/cloudpaymentsflutter/PaymentParameters.kt
package xdev.team.cloudpaymentsflutter

class PaymentParameters {
    var cardNumber:String?
    var cardDate:String?
    var cardCVV:String?
    var publicId:String?

    constructor(map: HashMap<String, String>){
        cardNumber = map["card_number"]
        cardDate = map["card_date"]
        cardCVV = map["card_cvv"]
        publicId = map["public_id"]
    }
}